eng/pipelines/aggregate-report.yml

```diff
@@ -0,0 +1,51 @@
+trigger: none
+pr: none
+
+jobs:
+  - job: Generate
+    variables:
+      - template: templates/variables/globals.yml
+    pool:
+      vmImage: 'windows-2019'
```

Comment:
Not a big issue, but if there's nothing Windows-specific, ubuntu-18.04 is much faster.

Reply:
Ok, AzureFileCopy is windows specific.